apps/fe/src/components/layout/header/index.tsx

```
@@ -341,11 +341,19 @@ export default function HeaderCom({ categoryMenu = [], regionMenu = [] }: Header
+                        <Button
+                            type="default"
+                            icon={<UserOutlined />}
+                            onClick={() =>
+                                router.push(
+                                    `${RouteNameEnum.LOGIN_PAGE}?to=${encodeURIComponent(
+                                        pathname,
+                                    )}`,
+                                )
+                            }
+                        ></Button>
+                    </Tooltip>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)